fs/remote/blob_test.go

```diff
@@ -58,30 +58,30 @@ import (
 const (
 	testURL            = "http://testdummy.com/v2/library/test/blobs/sha256:deadbeaf"
 	rangeHeaderPrefix  = "bytes="
-	sampleChunkSize    = 3
-	sampleMiddleOffset = sampleChunkSize / 2
+	sampleSpanSize     = 3
```

Comment:
As above, I think these are "blobs", not "spans".

Reply:
I updated this to keeping "chunk".